src/registry_schemas/example_data/schema_data.py

```diff
@@ -335,7 +337,8 @@
             'email': 'no_one@never.get',
             'date': '2020-02-18',
             'affectedFilings': [101, ],
-            'routingSlipNumber': '123456789'
+            'waveFees': True,
+            'priority': False
```

Comment:
The UI won't send 'priority' if 'waiveFees' is True, so a better example might be to omit the 'priority' property here.

Reply:
I concur, an example where the field is not present at all would be good to illustrate that it's not required.

Reply:
done